hedera-node/configuration/compose/application.properties

```diff
@@ -1,88 +1,20 @@
-### Bucket definitions
-hapi.throttling.buckets.slowOpBucket.capacity=8700000.0
-hapi.throttling.buckets.fastOpBucket.capacity=1300000.0
-hapi.throttling.buckets.fastOpBucket.overflow=slowOpBucket
-hapi.throttling.buckets.receiptsBucket.capacity=10000000.0
-### The desired network tps for CreateTopic is 5.0. With
-### a burstPeriod of 1.0, the bucket's
-###   capacity=tps * burstPeriod 
-### when split across 13 nodes, will yield a nodeCapacity 
-### of ~0.385, and all transactions will be throttled at 
-### a capacityRequired of 1.0. Thus we increase the burstPeriod 
-### to 2.6, yielding
-###   nodeCapacity=(5.0 * 2.6) / 13=1.0
-hapi.throttling.buckets.createTopicBucket.capacity=1300.0
-hapi.throttling.buckets.createTopicBucket.burstPeriod=2.6
-### Per-operation configs
-#### Transaction Service
-hapi.throttling.ops.transactionGetReceipt.bucket=receiptsBucket
-#### Consensus Service
-hapi.throttling.ops.consensusCreateTopic.bucket=createTopicBucket
-hapi.throttling.ops.consensusUpdateTopic.capacityRequired=1.67
-hapi.throttling.ops.consensusDeleteTopic.capacityRequired=1.67
-hapi.throttling.ops.consensusGetTopicInfo.capacityRequired=1.67
-hapi.throttling.ops.consensusSubmitMessage.capacityRequired=1.67
-#### Smart Contract Service
-hapi.throttling.ops.contractCall.bucket=slowOpBucket
-hapi.throttling.ops.contractCall.capacityRequired=669.0
-hapi.throttling.ops.contractCreate.bucket=slowOpBucket
-hapi.throttling.ops.contractCreate.capacityRequired=669.0
-hapi.throttling.ops.contractUpdate.bucket=slowOpBucket
-hapi.throttling.ops.contractUpdate.capacityRequired=669.0
-hapi.throttling.ops.contractDelete.bucket=slowOpBucket
-hapi.throttling.ops.contractDelete.capacityRequired=669.0
-hapi.throttling.ops.contractCallLocal.bucket=slowOpBucket
-hapi.throttling.ops.contractCallLocal.capacityRequired=669.0
-hapi.throttling.ops.contractGetInfo.bucket=slowOpBucket
-hapi.throttling.ops.contractGetInfo.capacityRequired=669.0
-hapi.throttling.ops.contractGetBytecode.bucket=slowOpBucket
-hapi.throttling.ops.contractGetBytecode.capacityRequired=669.0
-#### File Service
-hapi.throttling.ops.fileAppend.bucket=slowOpBucket
-hapi.throttling.ops.fileAppend.capacityRequired=669.0
-hapi.throttling.ops.fileCreate.bucket=slowOpBucket
-hapi.throttling.ops.fileCreate.capacityRequired=669.0
-hapi.throttling.ops.fileDelete.bucket=slowOpBucket
-hapi.throttling.ops.fileDelete.capacityRequired=669.0
-hapi.throttling.ops.fileUpdate.bucket=slowOpBucket
-hapi.throttling.ops.fileUpdate.capacityRequired=669.0
-hapi.throttling.ops.fileGetContents.bucket=slowOpBucket
-hapi.throttling.ops.fileGetContents.capacityRequired=669.0
-hapi.throttling.ops.fileGetInfo.bucket=slowOpBucket
-hapi.throttling.ops.fileGetInfo.capacityRequired=669.0
-
 # NODE-LOCAL/STATIC 
 accountBalanceExportEnabled=true
 accountBalanceExportDir=data/accountBalances/
 accountBalanceExportPeriodMinutes=10
 defaultListeningNodeAccount=0.0.3
-environment=1
+hedera.profiles.active=PROD
```

Comment:
Shall the ```hedera.profiles.active=DEV``` be better since the ```nettyMode``` is defaulted to be ```DEV```?

Reply:
For the Docker network we need `hedera.profiles.active=PROD` because when this is set to `DEV`, it makes only node `0.0.3` bind its gRPC server.

Apparently `nettyMode=DEV` just means to [accept all Netty's defaults](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/legacy/netty/NettyServerManager.java#L53) for config such as the keep-alive timeout, etc. instead of applying our own.

Perhaps this saves some resources when running locally? Not sure, I need to understand that config better. 😝

Reply:
@ljianghedera, @tinker-michaelj, we would want but we cannot make `hedera.profiles.active=PROD` when running locally because we have only 1 physical machine for the GRPC server. Docker compose solves this problem for running locally because it `simulates` a network of different (virtual) machines, allowing each of these virtual machines to have its own GRPC server.